lib/tlCore/StringFormat.h

```diff
@@ -4,6 +4,7 @@
 
 #include <string>
 #include <vector>
+#include <stdint.h>
```

Comment:
Could you change this to `<cstdint>`, I generally prefer to include the C++ versions of the headers.